unittests/Parser/ParserTests.cpp

```diff
@@ -608,10 +608,10 @@ cand %2 %3
 %2:i1 = eq 1:i32, %1
 cand %2 1:i1
 )i",
-      R"i(%0:i128 = var ; 0
-%1:i128 = bswap %0
```

Comment:
wait, why are you deleting an existing test?

Reply:
this looks like a typo from the conflicts i resolved from last time; will fix.